src/code/bank0.asm

```diff
@@ -5068,11 +5068,11 @@ LoadTitleScreenTiles::
 
     ldh  a, [hIsGBC]                              ; $2DBD: $F0 $FE
     and  a                                        ; $2DBF: $A7
```

Comment:
This checks whether `hIsGBC` is 0 or not, so the following tilesets were actually swapped around. This doesn't really matter because the two tilesets are identical in the official release, but for modders this can be confusing.